backend/src/test/java/trainingjournal/backend/service/PlanServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package trainingjournal.backend.service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.MockMvc;
+import trainingjournal.backend.model.Exercise;
+import trainingjournal.backend.repository.ExerciseRepository;
+import trainingjournal.backend.repository.PlanRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class PlanServiceTest {
+
+    PlanRepository planRepository = mock(PlanRepository.class);
+    ExerciseRepository exerciseRepository = mock(ExerciseRepository.class);
+    PlanService planService = new PlanService(planRepository, exerciseRepository);
+
+
+
+    @Test
+    void test_getAllExercises() {
```

Comment:
Namen (siehe oben)

Reply:
Und: Probier doch mal einen Test zu schreiben, sei es hier oder im Integrationstest wo tatsächlich auch Daten in der Liste enthalten sind